specmark.md

```diff
@@ -0,0 +1,353 @@
+---
+SPDX-License-Identifier: Community-Spec-1.0
+---
+
+# Markdown for SPDX 3 model specification
+
+The SPDX 3 model is written in a constrained subset of a Markdown markup
+language*, with predefined headings.
+
+Each model element (class, datatype, individual, property, and vocabulary)
+is defined in a distinct file.
+
+Specific headings and formatting are used to provide information for the
+generation of a machine-readable specification, in
+[Resource Description Framework (RDF)](https://en.wikipedia.org/wiki/Resource_Description_Framework)
+data model, by the [spec-parser](https://github.com/spdx/spec-parser/).
+
+For instance, a summary listed under the "Summary" heading will be represented
+as an `rdfs:comment` in the RDF file. Likewise, a value specified for the
+"minCount" of a property name under the "Properties" heading will be
+translated into a `sh:minCount` in the RDF file. See [an example](#example).
+
+Descriptions provided under the "Description" heading are intended for human
+reference and will not be incorporated into the RDF file.
+
+*The Markdown flavour used for the specification is
+[Python-Markdown](https://www.mkdocs.org/user-guide/writing-your-docs/#writing-with-markdown)
+as it is used by [MkDocs](https://www.mkdocs.org/) site generator.
+It differs slightly from
+[GitHub Flavored Markdown Spec](https://github.github.com/gfm/).
+
+## Model file content structure
+
+Each model file must adhere to a strict content structure:
+
+- All files must be encoded in UTF-8.
+- Each file must start with SPDX license information:
+  `SPDX-License-Identifier: Community-Spec-1.0`
+  and follows by one blank line.
+- The content immediately after the license information must begin with an
+  H1 heading containing the element's name.
+- Each element type has a predefined set of [allowed H2 headings](#syntax) and
+  labeled lists that must be used to structure its content.
+- There must be a blank line before and after a heading.
+- There must be a blank line before the beginning of a list.
+- There must be a blank line after the end of a list.
+- Any block level elements nested in a list, including paragraphs, sub-lists,
+  blockquotes, code blocks, etc., must always be indented by at least 4
```

Comment:
Why is that?

Reply:
Because the same file will be processed by MkDocs, and Python-Markdown that MkDocs use requires that.

https://python-markdown.github.io/#differences

Reply:
I have added a sentence clarify that in the doc. Thank you for flagging this up.